.github/workflows/advance-zed.yml

```diff
@@ -81,7 +81,10 @@ jobs:
         with:
           cache: yarn
           node-version-file: .node-version
-      - run: yarn add zed@brimdata/zed#${{ steps.zed_pr.outputs.sha }}
+      # The "git@github.com:" prefix causes Yarn to clone the repository
+      # rather than fetch a tarball.  This ensures "zed -version" and
+      # "zq -version" produce meaningful output.
+      - run: yarn add zed@git@github.com:brimdata/zed#${{ steps.zed_pr.outputs.sha }}
```

Comment:
I'm not sure about relying on ssh git, as it feels like github is slowly trying to get rid of it.

Reply:
I forgot that SSH authentication won't work in CI without some fiddling, so I went in a different direction and disabled the @yarnpkg/plugin-github plugin.

(I don't think SSH is on the way out, though. Perhaps you're thinking of the `git://` protocol, which they've [already disabled](https://github.blog/2021-09-01-improving-git-protocol-security-github/)?)